src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -1315,7 +1315,7 @@ static void _attachFont(LottieComposition* comp, LottieLayer* parent)
 }
 
 
-static bool _buildComposition(LottieComposition* comp, LottieGroup* parent)
+static bool _buildComposition(LottieComposition* comp, LottieLayer* parent)
```

Comment:
@hermet  is there a reason why this should be a LottieGroup object?

Reply:
@mgrudzinska No should be. we can change it if necessary.

Reply:
one static_cast less, if we keep Lottie Group I have to do 
```child->matteTarget = static_cast<LottieLayer*>(parent)->layerByIdx(child->mid);```